database/src/desktop/util_desktop_windows.cc

```diff
@@ -26,6 +26,42 @@ namespace firebase {
 namespace database {
 namespace internal {
 
+// Split a string based on specified character delimiter into constituent parts
+static std::vector<std::string> split_string(const std::string& s,
```

Comment:
If I'm reading this right, there is no difference between this and the above one. Should probably be moved to the shared util_desktop.cc/h

Reply:
Good call. This function went from a platform dependent `mkdir` to a simple string split that should be in a common location. Moved it.